### PR TITLE
Check Pangram in c++

### PR DESCRIPTION
**Pull request by ankan1811**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Program to check if a string is pangram or not

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

[#2158](https://github.com/TesseractCoding/NeoAlgo/issues/2158)

### CODE REVIEW

**Comment by ankan1811** [comment]
I have used simple hashing using map which is the simplest approach that can come to someone's mind !!! Please review it.

File: C-Plus-Plus/cp/check_pangram.cpp [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    if (result == 1)
+        cout << "Yes,the string is a pangram";
+    else
+        cout << "No,the string is not a pangram";
+
+    return 0;
+}
+
+/*
+	Time Complexity: O(N), where N is the length of the string.
```
Comment (HarshCasper) [inline C-Plus-Plus/cp/check_pangram.cpp:74]: Add sample I/O

**Review by HarshCasper — CHANGES_REQUESTED** [review]
One tiny change requested. Rest looks good to merge 👍

File: C-Plus-Plus/cp/check_pangram.cpp (hunk 1 above)
Comment (ankan1811) [reply C-Plus-Plus/cp/check_pangram.cpp:74]: @HarshCasper I have added now

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**